src/components/Admin/index.jsx

```diff
@@ -58,6 +58,18 @@ class Admin extends React.Component {
     );
   }
 
+  renderCsvErrorMessage() {
```

Comment:
CSV Button could probably be its own component, but for now I kept it in the admin page.

Reply:
Agreed. Should we create a ticket for this?